mitodl/mitxpro
# Redirect on no partial token

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #385 

#### What's this PR do?
Validates the request has a partial token and redirects to the start of the registration flow

#### How should this be manually tested?
- Verify registration works as normal.
- Go to /signup/details and /signup/extra and verify you're redirect to /signup
